iOS_SDK/OneSignalSDK/Source/OSMessagingController.m

```diff
@@ -311,6 +313,12 @@ - (void)onDidDismissInAppMessage:(OSInAppMessageInternal *)message {
     }
 }
 
+- (void)handleIAMPreview:(NSNotification *)nsNotification {
+    OSNotification *notification = [nsNotification.userInfo objectForKey:@"notification"];
```

Comment:
Do you think we should log that we are handling a preview IAM like we used to do?

Reply:
Good call, we do log already in the `OSNotificationsManager.handleIAMPreview` method that triggers this method here. That log happens before we post the notification to NSNotificationCenter. I think it is sufficient?

Reply:
Agreed that should be fine!